pagerduty/resource_pagerduty_event_orchestration_path_router.go

```diff
@@ -110,13 +112,15 @@ func resourcePagerDutyEventOrchestrationPathRouter() *schema.Resource {
 	}
 }
 
-func resourcePagerDutyEventOrchestrationPathRouterRead(d *schema.ResourceData, meta interface{}) error {
+func resourcePagerDutyEventOrchestrationPathRouterRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
Add contexrt methods to the API client, use RetryContext

Reply:
Addressed in https://github.com/PagerDuty/terraform-provider-pagerduty/pull/618/commits/9c7ddd0faf4d463f7dd1118d14936863a7232114